### PR TITLE
fix: another beta bump

### PR DESCRIPTION
**Pull request by nicolethoen**
## What
Closes #

## Type of change

- [ ] Feature
- [ ] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: packages/demo-app-ts/src/App.tsx [hunk 1]
```diff
@@ -167,8 +168,10 @@ class App extends React.Component<{}, AppState> {
               <BarsIcon />
             </PageToggleButton>
           </MastheadToggle>
-          <MastheadBrand>
-            <Brand src={imgBrand} alt="Patternfly Logo" heights={{ default: '40px' }} />
+          <MastheadBrand data-codemods>
```
Comment (jenny-s51) [inline packages/demo-app-ts/src/App.tsx:171]: Is this to be expected or can this be removed from here and `MastheadLogo`?
```suggestion
          <MastheadBrand>
```

Comment (nicolethoen) [reply packages/demo-app-ts/src/App.tsx:171]: good catch - i merged your suggestion!

File: packages/demo-app-ts/src/App.tsx [hunk 2]
```diff
@@ -167,8 +168,10 @@ class App extends React.Component<{}, AppState> {
...
+            <MastheadLogo data-codemods>
```
Comment (jenny-s51) [inline packages/demo-app-ts/src/App.tsx:172]:
```suggestion
            <MastheadLogo>
```

**Review by jenny-s51 — APPROVED** [review]
LGTM

**Pull request merged**